modules/roles_profiles/manifests/profiles/gecko_3_b_osx_1015_generic_worker.pp

```diff
@@ -0,0 +1,160 @@
+                        collect_cpu_time => false,
+                        ## If true, compute and report the sum of all non-idle CPU states.
+                        report_active    => false,
+                    },
+                    diskio   => {},
+                    procstat => {
+                        interval => '60s',
+                        exe      => 'generic-worker',
+                    },
+                },
+            }
+
+            class { 'talos':
+                user => 'cltbld',
+            }
+
+            $taskcluster_client_id    = lookup('generic_worker.datacenter_gecko_3_b_osx_1015.taskcluster_client_id')
+            $taskcluster_access_token = lookup('generic_worker.datacenter_gecko_3_b_osx_1015.taskcluster_access_token')
+            $livelog_secret           = lookup('generic_worker.datacenter_gecko_3_b_osx_1015.livelog_secret')
+            $quarantine_client_id     = lookup('generic_worker.datacenter_gecko_3_b_osx_1015.quarantine_client_id')
+            $quarantine_access_token  = lookup('generic_worker.datacenter_gecko_3_b_osx_1015.quarantine_access_token')
+            $bugzilla_api_key         = lookup('generic_worker.datacenter_gecko_3_b_osx_1015.bugzilla_api_key')
+
+            class { 'packages::zstandard':
+                version => '1.3.8',
+            }
+
+            class { 'generic_worker::multiuser':
+                taskcluster_client_id     => $taskcluster_client_id,
+                taskcluster_access_token  => $taskcluster_access_token,
+                worker_group              => $worker_group,
+                worker_type               => $worker_type,
+                data_dir                  => '/var/opt/generic-worker',
+                generic_worker_version    => 'v38.0.4',
+                generic_worker_sha256     => '5b97f98d52b97e2114b29ac42a0fcefb7b90ef70d1e24a3bbd6572a7ee6d4807',
+                taskcluster_proxy_version => 'v38.0.4',
+                taskcluster_proxy_sha256  => 'dd3095ee5aaa8c5fc017207f85bcb49fd83d429a759b07d82f6c0cb1c1b23fd7',
+                livelog_version           => 'v38.0.4',
+                livelog_sha256            => '2aef4bfbc214f1bece2154ee5ec5eedf9e495b562476873a759b229e25c3b448',
+                user                      => 'root',
+                #gw_dir                    => '/etc/generic-worker',
+            }
+
+            # exec { 'writes_in_catalina':
+            #     command => '/sbin/mount -uw /',
+            #     unless  => '/bin/test -d /builds || /bin/test -d /tools'
+            # }
+            #include dirs::tools
+
+            #include packages::google_chrome
+            #file { '/var/opt/generic-worker':
+            #    ensure => 'directory',
+            #    mode   => '0755',
+            #}
+
+            contain packages::nodejs
+            #contain packages::wget
+            contain packages::tooltool
+            file { '/tools/tooltool.py':
```

Comment:
For both Catalina and Bigsur, we need to stop setting up `/tools` and `/builds` since both OSes implement a RO `/` filesystem.  I know you can remount it RW but we really shouldn't.  I'm pretty sure Apple will prevent that in the future.

Reply:
I'm okay with moving these elsewhere, but isn't the synthetic.conf setup is simpler to then match the expectation of /tools etc on linux?

Reply:
I think we need to change that expectation.  It has always been poor form to assume you have write access to the root filesystem.

Reply:
Do you want to add a synthetic.conf to the bootstrap so that it can be used in bigsur also until the tasks are changed to not require a /tools ? Or do you want to remove this now, and that forces the change as part of greening-up the tasks to run on R8?